particlesystem/src/main/java/me/nikhilchaudhari/quarks/particle/Particle.kt

```diff
@@ -46,14 +48,22 @@ internal class Particle constructor(
     }
 
     fun show(drawScope: DrawScope) {
-        drawScope.drawArc(
-            color = color,
-            startAngle = 0f,
-            sweepAngle = 360f,
-            alpha = alpha,
-            topLeft = Offset(x, y),
-            size = Size(size, size),
-            useCenter = true
-        )
+        if(image != null){
+            drawScope.drawImage(
+                image = image,
+                alpha = alpha,
+                topLeft = Offset(x, y)
+            )
+        }else{
```

Comment:
Would you mind running ktlint once? 🙏🏼 
I haven't set anything up on this repo with github actions as this was just an experiment but I'll surely do that.

Reply:
Sure thing, I should have time this weekend to do it :)

Reply:
Ok I ran ktlint against this:
https://github.com/Smithyis1337/compose-particle-system/actions/runs/2646726589
 This was my first time using ktlint as I am a .net developer (just doing Kotlin for a personal project), so if its not right let me know 😃